app_dart/lib/src/request_handlers/update_timeseries.dart

```diff
@@ -52,8 +52,8 @@ class UpdateTimeSeries extends ApiRequestHandler<UpdateTimeSeriesResponse> {
     final ClientContext clientContext = authContext.clientContext;
     final KeyHelper keyHelper =
         KeyHelper(applicationContext: clientContext.applicationContext);
-    double goal = requestData[goalParam];
-    double baseline = requestData[baselineParam];
+    double goal = requestData[goalParam].toDouble();
+    double baseline = requestData[baselineParam].toDouble();
```

Comment:
Can these be null?  Should it be `requestData[goalParam]?.toDouble()`?  The current code would not throw if they were null, but it's not clear to me if we expect them to always be nonnull.

Reply:
They are expected to be non-null, since the ui doesn't allow to call api when any value is null. To be double safe, I updated accordingly.